### DISCUSSION TITLE
Styles for website widgets

### DISCUSSION THREAD

**Discussion by JankinHou** [chatwoot/chatwoot#9917] [Q&A]
After I add website inboxes to my website, the [start conversation] button looks like this.

![image](https://github.com/user-attachments/assets/4ac5f33e-2743-473c-9c72-960e6f49d490)
But I see it on the official website like this.
![image](https://github.com/user-attachments/assets/bf1348be-b6b1-43ea-a024-87c5d48aff74)

And the same is true on the management side of chatwoot.

![image](https://github.com/user-attachments/assets/4288b8be-a300-4b76-a945-a420c44105f7)

How can I set it to appear as a button?

**Comment by hncvj** [comment]
I'm looking for the same.

**Comment by immonsterx** [comment]
This has been bothering me for a long time. Have you solved it?
